packages/flutter/test/animation/animation_sheet_test.dart

```diff
@@ -40,8 +40,8 @@ void main() {
       const Duration(milliseconds: 100),
     );
 
-    final Widget display = await builder.display();
-    await tester.binding.setSurfaceSize(builder.sheetSize());
+    final Widget display = await builder.display(); // ignore: deprecated_member_use
```

Comment:
Can these just be migrated to the new API? If not, maybe leave a comment explaining the ignore?

Reply:
I plan to submit a new PR for the migration because the migration will break golden tests (because they'll be 1/3 in size), and in this way we're more sure what exactly caused the golden tests. It does look a bit redundant. Should I do them in the same PR?